lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/DocumentUtil.java

```diff
@@ -80,13 +80,19 @@ public static void writeDocToXmlFile(Document doc, File inputFile) throws Except
         transformer.setOutputProperty(OutputKeys.DOCTYPE_PUBLIC, "yes");
         doc.setXmlStandalone(true);
         DOMSource source = new DOMSource(doc);
-        StreamResult file = new StreamResult(new OutputStreamWriter(new FileOutputStream(inputFile), "UTF-8"));
-        transformer.transform(source, file);
+        try {
+            StreamResult file = new StreamResult(new OutputStreamWriter(new FileOutputStream(inputFile), "UTF-8"));
+            transformer.transform(source, file);
+        } catch (Exception ex) {
+            LOGGER.warning("Received exception during post processing of schema file " + inputFile.getAbsolutePath() + " : " + ex.getMessage());
+        }
     }
 
     private static TransformerFactory getTransformerFactory() throws TransformerConfigurationException {
         TransformerFactory transformerFactory = TransformerFactory.newInstance();
         transformerFactory.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, Boolean.TRUE);
+        transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
```

Comment:
Can you add a comment to these `setAttribute` that we are passing an empty String to disable that access? Since we are setting them to `false` everywhere else, it is not clear why we would pass empty String on these.

Reply:
Done